source/sync/migrate-sync-modes.txt

```diff
@@ -0,0 +1,356 @@
+  timeframe after the migration is finished.
+- Progress notifications will not work after migration because Flexible Sync does
+  not support them.  
+- Client apps that were connected to the Partition-Based Sync backend will 
+  experience a client reset after the backend is migrated to Flexible Sync. You 
+  should use a :ref:`client reset <handle-a-client-reset>` handler with recovery enabled.
+- Creating new tables in a migrated Flexible Sync backend using 
+  :ref:`Development Mode <development-mode>` from a Partition-Based Sync client
+  may lead to unexpected behavior.
+
+.. _realm-sync-migrate-stages:
+
+Migration Stages
+~~~~~~~~~~~~~~~~
+
+You can see your migration's stage in the App Services UI. There are three stages
+to a Sync mode migration:
+
+#. **Syncing Partition-Based Sync metadata** 
+   
+   At this stage, client apps can connect to your App's backend, but local writes 
+   will not sync. These writes will be lost if you don't have a 
+   :ref:`client reset handler <handle-a-client-reset>` to recover unsynced changes.
+	 
+   Clients can also receive updates to synced collections, but they won't
+   be able to receive upload acknowledgements.
+
+#. **Building Flexible Sync metadata** 
+   
+   Clients cannot connect at this stage. The duration of this stage is directly
+   proportional to the amount of data in synced collections.
+
+#. **Completed migration** 
+
+   :guilabel:`Manage Sync Migration` appears on the Device Sync page if the 
+   migration hasn't been committed or reverted.
+
+After the migration has finished, check out the :ref:`After Migration
+<realm-sync-migrate-after>` section for more information about how
+Partition-Based Sync clients interact with the Flexible Sync backend.
+
+Migrate Partition-Based Sync to Flexible Sync
+---------------------------------------------
+
+After beginning migration from Partition-Based Sync to Flexible Sync, the process
+is mostly automatic. However, you can control important parts of the migration.
+
+During a migration, you can cancel it. After a migration, you can revert it 
+or manually commit the migration.
+
+Begin Migration
+~~~~~~~~~~~~~~~
+
+Before you begin a migration, make sure you're aware of the
+:ref:`migration stages <realm-sync-migrate-stages>` and
+:ref:`effects <realm-sync-migrate-what-to-know>`.
+
+Migration generally has similar effects as :ref:`terminating <terminating-realm-sync>` 
+and re-enabling Device Sync. There are some additional effects of migration:
+
+- In the App Services UI, the Rules and Schema pages will be read-only.
+- Your App will experience a period of read-only Sync and then a 
+  period of downtime.
+- The new Flexible Sync configuration will set 
+  :ref:`client max offline time <client-maximum-offline-time>` to 30 days.
+- Storage in your Atlas cluster used by Device Sync metadata is temporarily doubled.
+  Partition-Based Sync metadata and Flexible Sync metadata must temporarily co-exist. 
```

Comment:
Is there any information about _how long_ this metadata must temporarily co-exist? i.e. is it for the client max offline time or some other definitive period, or is it variable?

Reply:
Good point. It only lasts as long as the migration itself. As soon as the migration is rolled back or completed, the doubled metadata is removed.

Reply:
Actually, @kmorkos, what happens if there isn't enough space in the cluster to double the metadata? Would this be a rare edge case?

Reply:
Ahhh, that makes perfect sense. Thank you!